devices/MAX1704x/samples/Program.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.MAX1704x;
+
+namespace MAX1704x.samples
```

Comment:
same here for the namespace even if it's just a sample.

Reply:
done